fronts-client/src/components/Editions/LastProofedIssueVersion/VersionPublicationTable.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import moment from 'moment';
+
+import { styled } from 'constants/theme';
+import { IssueVersionEvent } from 'types/Edition';
+
+const PublicationHistoryTable = styled.table`
```

Comment:
After testing this on CODE it looks like we no longer have an events table for publishing? Do we not want to keep this? It would help us know if an issue has been published as there is no indication within the Tool otherwise as far as I can tell.

Reply:
Re-instated Friday afternoon after discussion with Katy.